base/BUILD.gn

```diff
@@ -1103,10 +1112,6 @@ component("base") {
   # to provide the appropriate `#define` here.
   defines += [ "IS_RAW_PTR_IMPL" ]
 
-  if (!is_gold) {
-    defines += [ "ENABLE_TEST_DATA" ]
-  }
```

Comment:
Is this one not needed? Will base_unittests break in devel build?

Reply:
Yes it's needed, but it needs to be in a public config so that dependents also get the define, which is why I moved it to a config in the commit: [`c85e37d` (#2612)](https://github.com/youtube/cobalt/pull/2612/commits/c85e37d94082db2afa6058cb840ac4949f1b5806). It also was in a config in trunk.